MVCDemos/Views/Splitter/Overview.cshtml

```diff
@@ -16,7 +16,7 @@
                             .Resizable(true)
                             .Collapsible(true)
                             .MaxSize("75%")
-                            .Template(new JS("function(data){return paneContentTemplate(data, 'Central Pane')}"));
+                            .Template(new JS("function(data, itemIndex, itemElement){return paneContentTemplate(data, 'Central Pane', itemElement)}"));
```

Comment:
In jQuery you use the following arguments order 
<img width="298" alt="image" src="https://github.com/DevExpress/devextreme-demos/assets/15689080/b4ddb7d5-a357-450c-9c09-4f513eae3805">
Why you use order order here?

Reply:
ok i will change the order